Recipes/Recipes.API/HttpLoggingMiddleware.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Recipes.API;
```

Comment:
Could look tidier if we had a separate folder for Middlewares in the project. I doubt much logic will be checked, but the file structure is a simple one.